config/webpack/development.js

```diff
@@ -0,0 +1,5 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'development'
```

Comment:
all these need to be removed b/c you have a custom webpack config

future PR will put these in